app/script/media/MediaStreamHandler.js

```diff
@@ -241,7 +241,7 @@ z.media.MediaStreamHandler = class MediaStreamHandler {
    * @returns {Promise} Resolves when the input source has been replaced
    */
   replaceInputSource(mediaType) {
-    const isPreferenceChange = !this.mediaStreamInUse();
+    const isPreferenceChange = this.currentCalls.size === 0;
```

Comment:
Where is `size` coming from? Never seen it in the knockout documentation

Reply:
That's because it's a `Map` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map#Properties_2